front_right_controller:
  type: velocity_controllers/JointVelocityController
  joint: front_right_joint
  pid : {p: 1.0 , i: 0.0 , d: 0.0}

front_left_controller:
  type: velocity_controllers/JointVelocityController
  joint: front_left_joint
  pid : {p: 1.0 , i: 0.0 , d: 0.0}

  # Controller 3
rear_drive_controller:
    type: velocity_controllers/JointVelocityController
    joint: rear_wheel_joint
    pid : {p: 1.0 , i: 0.0 , d: 0.0}
 
/gazebo_ros_control:   
  pid_gains:
      front_right_joint: {p: 1.0, i: 0.0, d: 0.0}
      front_left_joint: {p: 1.0, i: 0.0, d: 0.0}
      rear_wheel_joint: {p: 1.0, i: 0.0, d: 0.0}